### ISSUE TITLE
Can I pass protein/gff file of closely related strain to augustus to annotate a novel strain?

### ISSUE THREAD

**Issue by Lamm-a** [Gaius-Augustus/Augustus#264]
I have the full Augustus annotations of a closely relates strain that had been trained through Braker using RNA-seq. Is it possible for me to use the output of that annotation as input for Augustus to annotate a strain of the same species?

**Comment by MarioStanke** [comment]
Yes. As the coordinates will be different, you have to use alignments. If the strains are very close, I suggest a nucleotide alignment:, a spliced alignment of the reference mRNAs or coding sequences against the new strain genome. Then make hints from the alignment.
Another approach is the CAT
pipeline https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit
It aligns the genomes and maps the coordinates this way (via TransMap).

**Comment by Lamm-a** [comment]
Thank you for the suggestions. How would you suggest I make hints from these alignments? sorry if this is a naive question, i am quite new to all of this.

**Comment by KatharinaHoff** [comment]
Please have a look at https://math-inf.uni-greifswald.de/storages/uni-greifswald/fakultaet/mnf/mathinf/stanke/augustus_wrp.pdf . This contains instructions on how to map predicted proteins to a related genome with GenomeThreader and create hints from that.
